h5p-bildetema-words-topic-image/tsconfig.json

```diff
@@ -7,7 +7,6 @@
     "./src/**/*",
     "typings.d.ts",
     "vite.config.ts",
-    ".storybook/**/*",
-    "../common/**/*"
```

Comment:
We are not importing `common` based on relative file paths, but instead as if it was a regular npm dependency. Including it in the tsconfig makes TypeScript go wild